index.html

```diff
@@ -0,0 +1,83 @@
+<html ng-app='resume'>
+
+<head>
+    <meta charset="UTF-8">
+    <title>Sergey Radkov Resume</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" />
+    <link href='https://fonts.googleapis.com/css?family=Marvel' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="asset/css/main.css">
+    <link rel="stylesheet" href="asset/css/about.css">
+    <link rel="stylesheet" href="asset/css/abilities.css">
+    <link rel="stylesheet" href="asset/css/experience.css">
+    <link rel="stylesheet" href="asset/css/portfolio.css">
+    <link rel="stylesheet" href="asset/css/contacts.css">
+
+    
+    
+</head>
+
+<body>
+
+    <div class="container-fluid">
+      <div class="row bg-full">               <!-- first page main picture-->
+        <div class='col-xs-12 col-md-6'>
+       </div>
+            
+
+<nav class="navbar navbar-default navbar-fixed-top col-md-2 col-md-offset-10">
+  <div class="container-fluid">
+    <!-- Brand and toggle get grouped for better mobile display -->
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar-collapse-exmpl" aria-expanded="false">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+    </div>
+
+    <!-- Collect the nav links, forms, and other content for toggling -->
+    <div class="collapse navbar-collapse" id="navbar-collapse-exmpl">
+      <ul class="navbar navbar-right" data-toggle="collapse" data-target="#navbar-collapse-exmpl">
+        
+        <li><a href="#about" >       ABOUT </a></li>
+        <li><a href="#abilities">   ABILITIES</a></li>
+        <li><a href="#experience">  RESUME</a></li>
+        <li><a href="#portfolio">   PORTFOLIO</a></li>
+        <li><a href="#contacts">    CONTACTS</a></li>
+        
+      </ul>
+    </div><!-- /.navbar-collapse -->
+  </div><!-- /.container-fluid -->
+</nav>
+
+      
+        
+            <div class='col-xs-7 col-xs-offset-1'>
+                <div  id='hello'>
+                        <h1>Sergey Radkov</h1>
+                        <hr>
+                        <h2>Interactive resume</h2>
+                </div>
+            </div>
+        </div>       
+    </div>
+     
+          
+    <about-me></about-me>
+    <abilities></abilities>
+    <experience></experience>
+    <portfolio></portfolio>
+    <contacts></contacts>
+
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+<script src='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js'></script>
```

Comment:
Although single quotes permitted by HTML spec https://www.w3.org/TR/html4/intro/sgmltut.html#h-3.2.2
It's a best practice to keep quotes consistent. Please, don't mix single and double quotes.